### ISSUE TITLE
Repo Not Reflecting Parse 1.17.3

### ISSUE THREAD

**Issue by HackShitUp** [parse-community/ParseLiveQuery-iOS-OSX#211]
When you update Parse 1.17.3, Parse Live Query returns the following error:
```
Cannot call value of non-function type 'ParseClientConfiguration'
```

Is this due to this?: https://github.com/parse-community/Parse-SDK-iOS-OSX/pull/1400

Here's a screenshot
<img width="974" alt="Error" src="https://user-images.githubusercontent.com/14321808/65347855-e76cc080-dbad-11e9-8d9c-d8783b49a112.png">

**Comment by noobs2ninjas** [comment]
It is. Our current release(2.6.0) supports Parse 1.16.0. Thats because 1.17.0+ is swift 5.0 and our current release is swift 4.2. That pull request was closed because it didnt do any of the things necessary to actually update the project properly. The pull request still being worked on that will update PLQ to version 2.7 is [here](https://github.com/parse-community/ParseLiveQuery-iOS-OSX/pull/210). Once that goes in our project will be officially swift 5 and all dependencies, including Parse, updated to most recent release. If you want to run Parse 1.17.x you certainly can, just auto fix the errors, but its official support is yet to come. 

Edit: Correction current release is 2.6.1 which supports Parse 1.17.0.

**Comment by TomWFox** [comment]
This should be resolved since #210, v2.7.0 has since been released.

**Comment by HackShitUp** [comment]
@TomWFox great, thanks tom
